src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,26 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private String name;
+    private String fuelType;
+    private int cylinderCount;
+    private int fuelConsumption;
+    private int horsePower;
 
+    public Engine(String name,
+                  String fuelType,
+                  int cylinderCount,
+                  int fuelConsumption,
+                  int horsePower) {
```

Comment:
this can be placed on two lines

there is no need to wrap each field onto a new line

Reply:
fixed